### PR TITLE
labhub: Adapt to the modified newcomer process

### PR DESCRIPTION
**Pull request by meetmangukiya**
- `invite me` variation of invite command is removed.
- "Hello world"s no longer trigger invites.
- Newcomer invitation message is changed to show the modification in the
  newcomer process.

Closes https://github.com/coala/corobo/issues/476

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: plugins/templates/labhub/promotions/newcomers.jinja2.md [hunk 1]
```diff
@@ -1,9 +1 @@
-Welcome @{{ target }}! :tada:
-
-We've just sent you an invite via email. Please accept it before proceeding forward.
```
Comment (jayvdb) [inline plugins/templates/labhub/promotions/newcomers.jinja2.md]: We still need an invite process via corobo, so this file and related code shouldnt be deleted.

**Comment by jayvdb** [comment]
unack https://github.com/coala/corobo/pull/486/commits/a87b74e499305371ef877f65aab4d97f603dffff

File: tests/labhub_test.py [hunk 2]
```diff
@@ -69,12 +69,11 @@ def test_hello_world_callback(self):
         labhub = testbot.bot.plugin_manager.get_plugin_obj_by_name('LabHub')
         labhub.TEAMS = teams
         self.mock_team.is_member.return_value = False
-        testbot.assertCommand('hello, world', 'newcomer')
+        testbot.assertCommand('hello, world', 'newcomer', timeout=10000)
```
Comment (meetmangukiya) [inline tests/labhub_test.py]: remove this timeout

**Comment by meetmangukiya** [comment]
Being done by @nvzard

**Pull request closed without merging**